index.html

```diff
@@ -0,0 +1,174 @@
+              <h3>@ millnium Hall , Addis Ababa</h3>
+          </div>
+        </div>
+        <img
+          class="friends"
+          src="assets/images/conferencethemenewImg.jpg"
+          alt="gitex conference theme."
+        />
+      </section>
+    </section>
+    <!-- Mid Section -->
+    <section class="main-program" id="program">
+      <h2>Main Events</h2>
+      <ul class="events">
+        <li class="event">
+          <img src="assets/images/program_icon_01 1.png" alt="keynote icon" />
+          <p class="event-title">Lecture</p>
+          <p class="event-desc">
+            Listen to the wizards from various continents about the ways of positive impacting 
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/jam.png" alt="" />
+          <p class="event-title">Workshop</p>
+          <p class="event-desc">
+            Try creating your own magic using open source power rather than just watching at it.
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/expo.svg" alt="Expo Icon" />
+          <p class="event-title">Expo</p>
+          <p class="event-desc">
+            A tour of the studio and showcase of unique items that make the show
+            unforgetable.
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/forum.svg" alt="" />
+          <p class="event-title">Forum</p>
+          <p class="event-desc">
+            Have the time to share your opinions with magicians for each topic
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/party.svg" alt="Party Icon" />
+          <p class="event-title">GetTog</p>
+          <p class="event-desc">
+            After the program, its gonna be get together and networking time
+            you dont wann miss it.
+          </p>
+        </li>
+      </ul>
+      <button class="get-ticket" type="button">Get your ticket Here</button>
+    </section>
+
+    <section class="guests" id="guests">
+      <h2>SPEAKERS</h2>
+      <div class="guests-list"></div>
+    </section>
+
+    <section class="partner" id="partners">
+      <h2>Partner</h2>
+      <ul class="partners">
+        
+        <li><img src="assets/images/netflix.png" alt="Hbo max" /></li>
+        <li><img src="assets/images/gitex-logo-full-new.png" alt="Netflix" /></li>
+        <li><img src="assets/images/Mastercard-logo.png" alt="Apple" /></li>
+        <li><img src="assets/images/OER Live Logo FINAL.webp" alt="Roke" /></li>
+        <li><img src="assets/images/gitex_logo_png-min.png" alt="Roke" /></li>
+      </ul>
+    </section>
+    <footer class="footer">
+      <!-- <a href="./index.html"> <img class="footer_logo" src="assets/images/GITEX+Africa+Website+Logo.jpg" alt="gitex logo" > </a> -->
+      <div class="rights">
+        <p>2023 Abenezer Addisu, All rights are Reserved</p>
+          <p>Built by ❤️ Abenezer</p>
+      </div>
```

Comment:
**I just want to say your project is looking 👌, great job!**
- [ ] Ouch! this is close but as part of the design specification you need to add a logo to the left of the text in your footer
- Behance design:
  ![image](https://user-images.githubusercontent.com/104968236/234905083-0c52118e-9ce5-441c-824b-0539dd347435.png)
- Your design:
  ![image](https://user-images.githubusercontent.com/104968236/234905403-0d573a92-cf59-4cf6-a629-4783665c142d.png)

- [ ] This applies to the about page as well. ✔